### PR TITLE
kernel: Add iommu_v2 driver

### PR DESCRIPTION
**Pull request by pprindeville**
Add a new IOMMU driver to support DPDK

### CODE REVIEW

**Comment by pprindeville** [comment]
cc: @zoki

**Comment by pprindeville** [comment]
Let's see if it builds on `master`...

**Comment by zokl** [comment]
> cc: @zoki

It is to me @pprindeville ? @zokl

**Comment by pprindeville** [comment]
> > cc: @zoki
> 
> It is to me @pprindeville ? @zokl

Yeah, sorry... sometimes this font is a little hard to read... eyes and els look alike...

File: package/kernel/linux/modules/virt.mk [hunk 1]
```diff
@@ -115,3 +115,34 @@ define KernelPackage/vfio-pci/description
...
 $(eval $(call KernelPackage,vfio-pci))
+
+
+define KernelPackage/iommu_v2
+  SUBMENU:=Virtualization
+  TITLE:=IOMMU Version 2 driver
+  KCONFIG:=\
+     CONFIG_UACCE=n \
+     CONFIG_IOMMU_DEBUGFS=n \
+     CONFIG_INTEL_IOMMU=y \
```
Comment (hauke) [inline package/kernel/linux/modules/virt.mk:126]: Isn't this option building it into the kernel?
Maybe we should activate iommu in the x86/64 target.

**Comment by chunkeey** [comment]
Merged. Thanks!

**Pull request closed without merging**